s30899-PatrykSzachniewicz/Kolokwium powtorzenie/zadania

```diff
@@ -0,0 +1,115 @@
+
+2.Masz taki uklad katalogow, jak wyzej i ciagle jestes w katalogu kota. Jak utworzyc poddrzewo 
+katalogow jan/kowalski w katalogu ale za pomoca jednej komendy?
+
+Podaj rozwiazania uzywajac sciezek absolutnych i relatywnych.
+
+Aby utworzyc taki uklad katalogow za pomoca sciezki bezwglednej skorzystamy z komendy:
+* mkdir -p /tmp/ukos/kot/ma/ale/jan/kowalski
+natomiast za pomoca sciezki wzglednej komenda bedzie nastepujaca:
+* mkdir -p ../../../kot/ma/ale/jan/kowalski
+
+3.Masz taki uklad katalogow, jak wyzej i ciagle jestes w katalogu kota. Jak przeniesc katalog ale do katalogu i uzywajac:
+-zrodlo (sciezka wzgledna), miejsce docelowe (sciezka absolutna)
+-zrodlo (sciezka absolutna), miejsce docelowe (sciezka wzgledna)
+
+W pierwszym przypadku uzyjemy komendy:
+* mv ../../../kot/ma/ale /tmp/ukos/ala/i
+natomiast w drugim komendy:
+* mv /tmp/ukos/kot/ma/ale ../../i
+
+4.Jak zamknac program, ktory nie reaguje na ctrl+c ?
+
+W przypadku kiedy program nie reaguje na ctrl+c a chcemy go zamknąć należy:
+* uzyc polecenia kill -9, aby wyslac sygnal kill do zamkniecia programu
+* wcisnac ctrl+z w celu wstrzymania programu i przeniesienia go do tla, aby nie obciazal procesora
+
+5.Jak wypisac wszystkie pliki w biezacym katalogu, ktorych nazwa zaczyna sie od al?
+
+Mozna zrobic to uzywajac na przyklad:
+* ls al*
+* find . -type f -name al*
+
+6a.Nadaj uprawnienia do katalogu ala tak aby:
+-kazdy mogl do niego wejsc
+-tylko grupa mogla wyswietlic co w nim jest
+-wlasciciel ma pelne uprawnienia
+
+HINT: Jakie sa uprawnienia poczatkowe?
+
+Podpowiedz: powyzsze 3 punkty opisu to jedno zadanie. Nie 3 osobne.
+Przygotuj 3 wariacje rozwiazania tego zadania uzywajac roznych notacji: 
+-oktalnej/osemkowej/cyfrowej
+-literowej ze znakiem rownosci
+-literowej z plusami i minusami
+
+Domyslne uprawnienia nadawane dla plikow to 666(rw-rw-rw-) natomiast dla katalogow to 777(rwxrwxrwx), jezeli ustawiona
+jest maska 'umask' to pliki i katalogi otrzymuja uprawnienia okreslone przez maske 'umask', jezeli ma on wartosc np. 022 
+to uprawnienia poczatkowe beda wygladac nastepujaca:
+* dla plikow 644(rw-r--r--)
+* dla katalogow 755(rwxr-xr-x)
+
+W celu nadania okreslonych uprawnien do katalogu ala, nalezy uzyc polecenia 'chmod' z wybrana notacja:
+* chmod 751 /tmp/ukos/ala
+* chmod u=rwx,g=rx,o=x /tmp/ukos/ala
+* chmod ugo+x,ug+r,u+w,go-w,o-r /tmp/ukos/ala
+
+6b.Jak korzystajac z notacji ze znakiem rownosci odebrac wszystkie uprawnienia do danego zasobu?
+
+W celu odebrania wszystkich uprawnien do danego zasobu, korzystajac z notacji ze znakiem rownosci, nalezy uzyc komendy np.:
+* chmod a=- plik
+
+7.Jak utworzyc plik z lista plikow w biezacym katalogu?
+Podaj rozwiazania uzywajace rozne techniki. Np nie uzywajac dedykowanego polecenia do listowania zawartosci katalogow.
+
+W celu utworzenia pliku z lista plikow w biezacym katalogu, mozna uzyc jedna z ponizszych komend:
+* ls > plik.txt
+* ls | tee plik.txt
+* ls -p | grep -v / | tee plik.txt
+* ls | dd of=plik.txt
```

Comment:
Wszystkie podane polecenia wykorzystują `ls` a chodziło o to by właśnie nie używać `ls`.
Ale powiedzmy, że możemy to zaliczyć.